src/index.js

```diff
@@ -151,6 +151,7 @@ export { createShader, createShaderFromCode } from './scene/shader-lib/utils.js'
 export { ProgramLibrary } from './scene/shader-lib/program-library.js';
 export { shaderChunks } from './scene/shader-lib/chunks/chunks.js';
 export { shaderChunksLightmapper } from './scene/shader-lib/chunks/chunks-lightmapper.js';
+export { ChunkBuilder } from './scene/shader-lib/chunk-builder.js';
```

Comment:
is this needed by the Shed or elsewhere?

Reply:
Yep shed. Probably ok to export?

Reply:
yeah I don't see why not. Perhaps add a comment to that file that its used by Shed, so that we don't just modify it considering its private.